### PR TITLE
[Governance] Merge voting status and execution status to one status

### PR DESCRIPTION
**Pull request by zihan-aptos**
* Merge `ProposalVotingState` and `ProposalExecutionState` to `ProposalStatus` that represents the general proposal satatus.
* Update the new status for proposals table and proposal page.
* Add a tooltip on hover to explain the meaning of status. **_Please advice on the content!!!_**
Screen Shot 2022-08-19 at 12 19 48 PM
Screen Shot 2022-08-19 at 12 19 37 PM

### CODE REVIEW

File: src/pages/Governance/components/ProposalStatusTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        ProposalStatus.FAILED,
+        "- Proposal failed because there were not enough votes.",
+      )}
+      {StatusBox(
+        ProposalStatus.REJECTED,
+        "- Proposal is rejected by the majority of voters.",
+      )}
+      {StatusBox(
+        ProposalStatus.AWAITING_EXECUTION,
+        "- Proposal is passed by by the majority of voters and waiting for execution.",
```
Comment (msavir) [inline src/pages/Governance/components/ProposalStatusTooltip.tsx:41]:
```suggestion
        "- Proposal is passed by the majority of voters and waiting for execution.",
```

File: src/pages/Governance/constants.tsx [hunk 2]
```diff
@@ -12,6 +12,8 @@ export const negativeColorOnHover: string = "#dd6767";
 export const negativeColorWithOpacity: string = "rgba(249,115,115,0.1)";
 // yellow
 export const warningColor: string = "#D8BF45"; // yellow
+// grey
+export const secondaryColor: string = grey[400]; // grey
```
Comment (msavir) [inline src/pages/Governance/constants.tsx:16]: should we support theme mode? 
````${theme.palette.mode === "dark" ? grey[100] : grey[400]}````

File: src/pages/Governance/utils.tsx [hunk 3]
```diff
@@ -29,7 +34,7 @@ function canBeResolvedEarly(proposal: Proposal): boolean {
 
 // replicate on-chain logic get_proposal_state()
 // https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/voting.move
-export function getProposalState(proposal: Proposal): ProposalVotingState {
+function getProposalVotingState(proposal: Proposal): ProposalVotingState {
```
Comment (msavir) [inline src/pages/Governance/utils.tsx:37]: nit: `ProposalVotingState` type can also be changed to `ProposalState`

**Review by msavir — APPROVED** [review]

File: src/pages/Governance/constants.tsx (hunk 2 above)
Comment (zihan-aptos) [reply src/pages/Governance/constants.tsx:16]: I tried it on light mode too, grey[400] works good on both themes, so I might just keep it.

**Pull request merged**